plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -216,6 +251,9 @@ export class EventPipelineRunner {
             event.team_id
         )
 
+        // TRICKY: old client might still be sending heatmap_data as passengers on other events
+        // so this step is here even though up-to-date clients will be sending heatmap events
+        // for separate processing
```

Comment:
I'd love to just turn heatmaps off for old SDK versions, as in we just drop the data in the non-heatmap pipeline in a month or so (potentially emit ingestion warning about SDK update needed, but maybe not even as it's a rather fresh product)